dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -154,7 +155,7 @@ default VersionTypeValidation checkValidJsonAndYamlFiles(Set<SourceFile> sourcef
         Map<String, String> validationMessageObject = new HashMap<>();
         for (SourceFile sourcefile : sourcefiles) {
             if (Objects.equals(sourcefile.getType(), fileType)) {
-                Yaml yaml = new Yaml();
+                Yaml yaml = new Yaml(new SafeConstructor());
                 try {
                     yaml.load(sourcefile.getContent());
```

Comment:
In other cases, you read content with `new Yaml(new SafeConstructor())`, then if it doesn't error, you use the default `new Yaml()`. Is it safe to it differently here? Not saying it's not, just sanity checking.

Reply:
I think this is the preferred pattern. I used the other pattern in places where I couldn't just instantiate like this (without breaking tests).